src/pybind/mgr/dashboard/controllers/auth.py

```diff
@@ -94,20 +94,26 @@ def create(self, username, password, ttl: Optional[int] = None):
                     multicluster_config = Settings.MULTICLUSTER_CONFIG.copy()
                 try:
                     if fsid in multicluster_config['config']:
-                        existing_entries = multicluster_config['config'][fsid]
-                        if not any((entry['user'] == username or entry['cluster_alias'] == 'local-cluster') for entry in existing_entries):  # noqa E501 #pylint: disable=line-too-long
-                            existing_entries.append({
+                        cluster_configurations = multicluster_config['config'][fsid]
+                        for config_item in cluster_configurations:
+                            if config_item['user'] == username or config_item['cluster_alias'] == 'local-cluster':  # noqa E501  #pylint: disable=line-too-long
+                                config_item['token'] = token  # Update token
+                                break
```

Comment:
checking this again, how's this related to the multi-site progress?

Reply:
we need to store the local cluster's token as well, lets suppose you have multi-cluster setup and you want to perform replication from cluster2 - local-cluster, in that case we switch to cluster2 from local and open the replication wizard, from there we can select local cluster from a list of clusters and do the replication